learn.javascript.ru/bitwise-operators/task_02/alert.js

```diff
@@ -0,0 +1,10 @@
+// Script for task #1 from https://learn.javascript.ru/bitwise-operators
+
+
+function isInteger(int) {
+  return (int === (int^0));

```

Comment:
what is int^0? Can you share the link or explain what is and how it is work? Because I see this at first time

Reply:
oh, I found. Hmm, okay, but will you use such approach on production? Seems too complicated for simple developers)) But, I am OK with that.